### PR TITLE
editing new lessons

### PR DESCRIPTION
**Pull request by CaitlinBate**
copy editing

renumbering

formatting

adding code for other nodes (actel)

### CODE REVIEW

File: _episodes/11-act-actel.md [hunk 1]
```diff
@@ -112,14 +114,13 @@ all_stations <- bind_rows(deploy_metadata, otn_deploy_metadata)
 
 # Mutate metadata into Actel format ----
 
-# Create a station entry from the projectcode and station number.
+# Create a station entry from the glatos array and station number.
```
Comment (naomitress) [inline _episodes/11-act-actel.md:117]: did you mean glatos array here? ACT array?

File: _episodes/11-act-actel.md [hunk 2]
```diff
@@ -130,16 +131,19 @@ full_receiver_meta <- all_stations %>%
 As we saw earlier, tagging metadata is entered into Actel as `biometrics`, and deployment metadata as `deployments`. These data structures also require a few specially named columns, and a properly formatted date.
 ~~~
 # All dates will be supplied to Actel in this format:
-actel_datefmt = '%Y-%m-%d %H:%M:%S'
+actel_datefmt = '%Y-%m-%d %H:%M:%S
```
Comment (naomitress) [inline _episodes/11-act-actel.md:134]: a single quote got dropped?

File: _episodes/11-act-actel.md (hunk 1 above)
Comment (CaitlinBate) [reply _episodes/11-act-actel.md:117]: copied from Bruce's code directly here, but i assume it should be projectcode. ill fix in all the spots

File: _episodes/11-act-actel.md (hunk 2 above)
Comment (CaitlinBate) [reply _episodes/11-act-actel.md:134]: good catch

File: _episodes/11-act-actel.md [hunk 3]
```diff
@@ -259,10 +289,297 @@ actel_explore_output <- explore(datapack=actel_project,
...
+
+~~~
+{: .language-r}
+
+Can we design a graph and write it into spatial.txt that fits all these Arrays together? The station value we put in Array for our PROJ61 and PROJ60 projects looks to be a bit too granular for our purposes. Maybe we can combine many arrays that are co-located in open water into a singular 'zone', preserving the complexity of the river systems but creating a large basin to which we can connect the furthest downstream of those river arrays.
+
+To do this, we only need to update the arrays in our spatial.csv file or actel_spatial dataframe. We don't need to edit our source metadata! We will have to define a spatial.txt file and how these newly defined Arrays interconnect. While there won't be time to do that for this example dataset and its large and very complicated region, this approach is definitely suitable for small river systems and even perhaps for multiple river systems feeding a bay and onward to the open water. If you'd like to apply Actel to your data and want to define a custom spatial.txt file, there are some code examples included in code/day2/5_actel_custom_spatial.R that might be helpful to get you started.
+
+## MIGRAMAR Node
```
Comment (naomitress) [inline _episodes/11-act-actel.md:331]: this gets cut?

Comment (naomitress) [reply _episodes/11-act-actel.md:331]: or not til after the fork

Comment (CaitlinBate) [reply _episodes/11-act-actel.md:331]: unclear, depends on testing of the 5_actel_custom_spatial.R file i think?

File: _episodes/14-animation-with-pathroutr.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+---
+
+Basic animations using `gganimate` are great for many purposes but you will soon run into issues where your fish tracks are moving across land barriers, especially in more complex environments. This is because the `geom` used in the previous lesson choose the most direct path between two detection points. To avoid this we need to use a specific land avoidance tool. For our next animation we will use the `pathroutr` package which you can find out more about [here](https://jmlondon.github.io/pathroutr/). 
+
+We will begin in much the same way we did for our basic animation lesson with getting our data ready, but things will differ when we start to use `pathroutr` since there are specific data formats expected.
+
+
+### Preparing our Data 
+
+We will start out in the same way we did for the basic animation. We will use the same assumptions about the data this time and only look at one fish. We will also filter down the data and only look at **5 detection events** as an example subset due to the computational intensity of `pathroutr` and its calculations.
```
Comment (naomitress) [inline _episodes/14-animation-with-pathroutr.md:17]: this sentence is identical to the one on line 12

**Review by naomitress — APPROVED** [review]

**Pull request merged**